src/views/Accessibility/AccessibilityRequestDetailPage.test.tsx

```diff
@@ -52,120 +56,163 @@ describe('AccessibilityRequestDetailPage', () => {
       }
     }
   ];
-  it('renders without crashing', () => {
-    const wrapper = shallow(
-      <MemoryRouter>
-        <MessageProvider>
-          <AccessibilityRequestDetailPage />
-        </MessageProvider>
+
+  it('renders without crashing', async () => {
+    render(
+      <MemoryRouter
+        initialEntries={['/508/requests/e0a4de2f-a2c2-457d-ac08-bbd011104855']}
+      >
+        <MockedProvider mocks={defaultMocks} addTypename={false}>
+          <MessageProvider>
+            <Provider store={store}>
+              <Route path="/508/requests/:accessibilityRequestId">
+                <AccessibilityRequestDetailPage />
+              </Route>
+            </Provider>
+          </MessageProvider>
+        </MockedProvider>
       </MemoryRouter>
     );
-    expect(wrapper.find('AccessibilityRequestDetailPage').length).toEqual(1);
+
+    await waitForElementToBeRemoved(() => screen.getByTestId('page-loading'));
+
+    expect(
+      screen.getByTestId('accessibility-request-detail-page')
```

Comment:
Generally we don't want to use `getByTestId`. This situation is okay because there's no accessibility that needs to be tested. We're just making sure the view is mounted.